java/src/main/java/ai/rapids/cudf/ParquetOptions.java

```diff
@@ -56,6 +68,41 @@ public Builder withTimeUnit(DType unit) {
       return this;
     }
 
+    /**
+     * Include one or more specific columns.  Any column not included will not be read.
+     * @param names the name of the column, or more than one if you want.
+     */
+    public Builder includeColumn(String... names) {
+      for (String name : names) {
```

Comment:
Can we just call super(names) here, and then loop for just binaryAsStringColumns.  Same for the other includeColumn below?